src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -330,19 +336,20 @@ public WebDriver createWebDriver(TestCleaner cleaner, TestName testName, Elastic
         cleaner.addTask(new Statement() {
             @Override
             public void evaluate() {
-                String browser = System.getenv("BROWSER");
-                if(browser == null || browser.equals("firefox")) {
-                    //https://github.com/mozilla/geckodriver/issues/1151
-                    //https://bugzilla.mozilla.org/show_bug.cgi?id=1264259
-                    //https://bugzilla.mozilla.org/show_bug.cgi?id=1434872
-                    d.navigate().to("about:mozilla");
-                    Alert alert = ExpectedConditions.alertIsPresent().apply(d);
-                    if (alert != null) {
-                        alert.accept();
-                        d.navigate().refresh();
-                    }
+                switch(getBrowser()) {
+                    case "firefox":
```

Comment:
we likely to get any more firefoxes? i.e. should a contains be done?

Reply:
I don't really know... In any way it's also done like this in `FallbackConfig#createWebDriver` so both places would need to be updated.